pandas/tests/indexes/timedeltas/test_setops.py

```diff
@@ -73,3 +74,80 @@ def test_intersection_bug_1708(self):
         result = index_1 & index_2
         expected = timedelta_range('1 day 01:00:00', periods=3, freq='h')
         tm.assert_index_equal(result, expected)
+
+    @pytest.mark.parametrize("sort", [None, False])
+    def test_intersection_equal(self, sort):
+        # for equal indicies intersection should return the original index
+        first = timedelta_range('1 day', periods=4, freq='h')
+        second = timedelta_range('1 day', periods=4, freq='h')
+        intersect = first.intersection(second, sort=sort)
+        if sort is None:
+            tm.assert_index_equal(intersect, second.sort_values())
+        assert tm.equalContents(intersect, second)
+
+        # Corner cases
+        inter = first.intersection(first, sort=sort)
+        assert inter is first
+
+    @pytest.mark.parametrize("sort", [None, False])
```

Comment:
parametrize this

Reply:
What do you mean exactly? (as you are commenting this on a 'parametrize' line)

Reply:
Is that and the same comments below meant as a way to parameterize the index ranges in the tests? I can understand it in the tests below but not sure what you mean in this case.

Reply:
yes same comment as above, as the cases in the parameterization (in addition to this one)